data/language/en-GB.txt

```diff
@@ -3801,3 +3797,9 @@ STR_6739    :{WINDOW_COLOUR_2}Date: {BLACK}{STRINGID}
 STR_6740    :{WINDOW_COLOUR_2}Cash: {BLACK}{CURRENCY2DP}
 STR_6741    :{WINDOW_COLOUR_2}Num. rides: {BLACK}{UINT16}
 STR_6742    :{WINDOW_COLOUR_2}Num. guests: {BLACK}{UINT16}
+STR_6743    :{WINDOW_COLOUR_2}Guests’ intensity preference:
+STR_6744    :No preference
+STR_6745    :Balanced
```

Comment:
I wonder if 'Randomised' would be a better label, considering that's actually what's happening: random values by guest, instead of _fixed_ values for _all_ (new) guests.

Reply:
Just because something is random doesn't mean it is not balanced, that depends on the distribution so I think the label is fitting. When people see random they might get the impression that it will be chaotic.

Reply:
That's a good argument, yes. For the record, the game will default to 'balanced'.